install_spot_ros2.sh

```diff
@@ -1,11 +1,11 @@
-ARM=false
+ARCH="amd64"
```

Comment:
I think it would make sense to have a `VERSION="3.3.2"` and then use that through out as well. Likely will save us from a typo in the future.

Reply:
Good idea. Done in https://github.com/bdaiinstitute/spot_ros2/pull/234/commits/34a597492901413d8038cf173e59425bcee1a832.